pages/use-solid.md

```diff
@@ -35,15 +35,22 @@ The following table lists some Pod Provider:
 | [solidweb.org](https://solidweb.org)               | [Solid Grassroots](https://gitlab.com/groups/solidweb.org/-/group_members) | [Hosteurope](https://www.hosteurope.de)       | France          | [NSS](https://github.com/solid/node-solid-server)           |
+| [dev.inrupt.net](https://dev.inrupt.net/)          | [Inrupt, Inc.](https://www.inrupt.com)                                     | [Amazon](https://aws.amazon.com)              | USA              | [NSS](https://github.com/solid/node-solid-server)           |
 | [trinpod.us](https://trinpod.us)                   | [Graphmetrix, Inc.](https://graphmetrix.com/terms)                         | [Amazon](https://aws.amazon.com)              | USA              | [TrinPod](https://graphmetrix.com/construction/trinpod)     |
-| [use.id](https://get.use.id/)                      | [Digita](https://www.digita.ai/)                                           | [DigitalOcean](https://digitalocean.com/)              | EU              | [CSS](https://github.com/CommunitySolidServer/CommunitySolidServer)     |
-| [solidweb.me](https://solidweb.me/)                | [Meisdata](https://github.com/serverproject-dev)                           | [Hosteurope](https://www.hosteurope.de)        | France              | [CSS](https://github.com/CommunitySolidServer/CommunitySolidServer)        |
-| [Data Pod](https://datapod.igrant.io/)                | [iGrant.io, Sweden](https://igrant.io/)                           | [RedPill Linpro](https://www.redpill-linpro.com/en), [AWS](https://aws.amazon.com/), [GCP](https://cloud.google.com/) or [Azure](https://azure.microsoft.com/en-in)        | EU              | [NSS](https://github.com/solid/node-solid-server)        |
-| [redpencil.io](https://solid.redpencil.io/)                | [redpencil.io](https://redpencil.io/)                           | [Hetzner](https://www.hetzner.com/)        | Finland              | [CSS](https://github.com/CommunitySolidServer/CommunitySolidServer)        |
-| [teamid.live](https://teamid.live/)                | [Meisdata](https://github.com/serverproject-dev)                           | [Hosteurope](https://www.hosteurope.de)        | France              | [CSS](https://github.com/CommunitySolidServer/CommunitySolidServer)        |
+| [trinpod.eu](https://trinpod.eu)                   | [Graphmetrix, Inc.](https://graphmetrix.com)                               | [Hetzner](https://www.hetzner.com/)           | Finland              | [TrinPod](https://graphmetrix.com/construction/trinpod)     |
+| [use.id](https://get.use.id/)                      | [Digita](https://www.digita.ai/)                                           | [DigitalOcean](https://digitalocean.com/)     | EU              | [CSS](https://github.com/CommunitySolidServer/CommunitySolidServer)     |
```

Comment:
Can't tell exactly if this is just an identity provider and/or storage, and also whether it is for individuals and/or business. But okay.